test/test_integration.rb

```diff
@@ -240,6 +240,7 @@ def test_restart_closes_keepalive_sockets_workers
   end
 
   def test_restart_with_prune_bundler_keeps_bundler_env
+    skip NO_FORK_MSG unless HAS_FORK
```

Comment:
Do this test actually need to use cluster mode/forking? If we don't need to, we should remove it and allow the test to run on JRuby.